### PR TITLE
Popover z-index

### PR DESCRIPTION
**Pull request by elanoism**
Add some basic z-index to make the Popover usable in real environment (**ze-d**).

### CODE REVIEW

File: src/components/AutoCompleteInput/AutoCompleteInput.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ const ItemsWrapper = styled('div')`
   height: 0px;
   overflow: visible;
   margin-top: ${props => props.theme.spacings.bit};
+  z-index: 20;
```
Comment (connor-baer) [inline src/components/AutoCompleteInput/AutoCompleteInput.js]: Can you add this value to the theme, please? https://github.com/sumup/circuit-ui/blob/master/src/themes/default.js#L228

File: src/components/Popover/Popover.js [hunk 2]
```diff
@@ -122,7 +122,10 @@ class Popover extends Component {
         >
           {({ ref, style }) =>
             isOpen && (
-              <div style={style} ref={this.receivePopoverRef}>
+              <div
+                style={{ ...style, zIndex: '10' }}
```
Comment (connor-baer) [inline src/components/Popover/Popover.js]: Meh, I'm not too happy about the style attribute. Any way of making this a styled component?

Comment (fernandofleury) [reply src/components/Popover/Popover.js]: you can also use the `css` fn is really needed.

File: src/components/AutoCompleteInput/AutoCompleteInput.js (hunk 1 above)
Comment (elanoism) [reply src/components/AutoCompleteInput/AutoCompleteInput.js]: Oh, I wasn't aware we already added this.  Thanks.

File: src/components/Popover/Popover.js (hunk 2 above)
Comment (elanoism) [reply src/components/Popover/Popover.js]: Styled component for the `zIndex` - yes, but the style in this case is dynamic in nature and can be updated too frequently in some situations, which can have performance implications.

Comment (elanoism) [reply src/components/Popover/Popover.js]: @connor-baer

File: src/components/AutoCompleteInput/AutoCompleteInput.js (hunk 1 above)
Comment (fernandofleury) [reply src/components/AutoCompleteInput/AutoCompleteInput.js]: I'm always one step ahead

**Pull request merged**